model_specific/SM/sm_mw.cpp

```diff
@@ -0,0 +1,84 @@
+// ====================================================================
+// This file is part of FlexibleSUSY.
+//
+// FlexibleSUSY is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published
+// by the Free Software Foundation, either version 3 of the License,
+// or (at your option) any later version.
+//
+// FlexibleSUSY is distributed in the hope that it will be useful, but
+// WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with FlexibleSUSY.  If not, see
+// <http://www.gnu.org/licenses/>.
+// ====================================================================
+
+#include "sm_mw.hpp"
+#include "logger.hpp"
+#include <cmath>
+
+namespace flexiblesusy {
+namespace sm_mw {
+
+namespace {
+
+constexpr double sqr(double x) noexcept { return x*x; }
+
+} // anonymous namespace
+
+/** 
+ * Calculates the prediction for the W boson mass in the Standard
+ * Model, using the fit formula Eq (45) from arXiv:1411.7040.
+ * 
+ * @param mh SM Higgs boson pole mass
+ * @param mt SM top quark pole mass
+ * @param as MS-bar alpha_s(MZ) in the SM with 5 quark flavours
+ * @param da5had hadronic contributison Delta alpha_{had}^{(5)}(MZ^2)
+ *
+ * The authors of arXiv:1411.7040 recommend used (Table 1):
+ *
```

Comment:
What does this comment mean?  We are not recommending they use these values from 2014 are we?  Maybe 'recommend' is some pasted accident and should just be deleted?

Reply:
With the comment I just wanted to give a hint on a possible input parameter set. I find that this sometimes help to get some first example working (and refine/update it later). Anyway, we are not recommending to use these values. These are just the values the authors of arXiv:1411.7040 used.

I can remove it, if you like.

Reply:
I've removed the statement in commit 08de09bd655b2ed39cd83739e2c34f7e7c8558a9